machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="cpmso-troubleshooting"]
+= Troubleshooting the Control Plane Machine Set Operator
```

Comment:
This section is commented out in the build and must be reviewed in a later PR that finishes this section.